modules/performance.nix

```
@@ -54,32 +59,41 @@ in
     };
   };
 
-  config.performance = {
-    # Set option value with default priority so that values are appended by default
-    combinePlugins.pathsToLink = [
-      # :h rtp
-      "/autoload"
-      "/colors"
-      "/compiler"
-      "/doc"
-      "/ftplugin"
-      "/indent"
-      "/keymap"
-      "/lang"
-      "/lua"
-      "/pack"
-      "/parser"
-      "/plugin"
-      "/queries"
-      "/rplugin"
-      "/spell"
-      "/syntax"
-      "/tutor"
-      "/after"
-      # ftdetect
-      "/ftdetect"
-      # plenary.nvim
-      "/data/plenary/filetypes"
+  config = {
+    # FIXME: Ugly fix for:
+    # vimplugin-plugin-pack/lua/telescope/config.lua:1: module 'plenary.strings' not found
+    # The logic itself has to be properly fixed.
+    extraPlugins = lib.mkIf config.performance.combinePlugins.enable [
+      pkgs.vimPlugins.plenary-nvim
     ];
```

Could we avoid the large formatting diff by doing: 


```suggestion
    config.extraPlugins = lib.mkIf config.performance.combinePlugins.enable [
      pkgs.vimPlugins.plenary-nvim
     ];
```

---

Also, note that this only fixes `combinePlugins` for plugins that depend on plenary. If a user "combines" some other plugin we don't test for, with a different transitive lua dependency, it will still break for them.